### ISSUE TITLE
Sample Packing

### ISSUE THREAD

**Issue by trolommonm** [EvolvingLMMs-Lab/LLaVA-OneVision-1.5#59]
I understand that with sample packing, it reduces padding and improves GPU utilization. However, less optimization steps are taken. Have you guys done any experiments on how this may affect the training convergence? For e.g. with sample packing, do you notice that your final loss is higher than without sample packing?

**Comment by trolommonm** [comment]
Were there any learning rate (or any other hyperparameter) changes that you had to do?
